css/style.css

```diff
@@ -845,17 +854,11 @@ button:hover {
 
 @media (max-width: 480px) {
 	/* hide board actions on mobile */
-	#board-actions {
+	#board-header-controls {
 		display: none !important;
```

Comment:
This is a serious deal-breaker for mobile usability. Why don't we move these actions into a ... `.icon-more` popover menu?